rickshaw-settings.json

```diff
@@ -20,13 +20,13 @@
         }
     },
     "workshop": {
-        "force-builds": "false",
-        "refresh-timestamps": {
+        "force-builds": "false"
+    },
+    "quay": {
+        "image-expiration": "2w",
+        "refresh-expiration": {
             "token-file": null,
```

Comment:
if this is just a file path, what is the problem in setting the value ?

Reply:
Not sure what you mean by this.